python/hail/tests.py

```diff
@@ -698,17 +691,20 @@ def test_range(self):
         ds = hl.utils.range_matrix_table(100, 10)
         self.assertEqual(ds.count_rows(), 100)
         self.assertEqual(ds.count_cols(), 10)
-        et = ds.annotate_entries(entry_idx = 10 * ds.row_idx + ds.col_idx).entries_table().index()
+        et = ds.annotate_entries(entry_idx=10 * ds.row_idx + ds.col_idx).entries_table().index()
```

Comment:
really? space even when rhs is a longer expression? looks weird  to me

Reply:
this was an autoformat change, I reverted it.